plugins/modules/rds_global_cluster_info.py

```diff
@@ -0,0 +1,197 @@
+                description: The status of write forwarding for a secondary cluster in the global cluster.
+                type: str
+                sample: disabled
+    failover_state:
+        description:
+        - A data object containing all properties for the current state of an in-process or
+          pending switchover or failover process for this global cluster (Aurora global database).
+        - This object is empty unless the SwitchoverGlobalCluster or FailoverGlobalCluster operation was called on this global cluster.
+        type: dict
+        contains:
+            status:
+                description:
+                - The current status of the global cluster.
+                type: str
+                sample: "pending"
+            from_db_cluster_arn:
+                description: The Amazon Resource Name (ARN) of the Aurora DB cluster that is currently being demoted, and which is associated with this state.
+                type: str
+                sample: arn:aws:rds:us-east-1:123456789012:cluster:ansible-test-primary
+            to_db_cluster_arn:
+                description: The Amazon Resource Name (ARN) of the Aurora DB cluster that is currently being promoted, and which is associated with this state.
+                type: str
+                sample: arn:aws:rds:us-east-2:123456789012:cluster:ansible-test-secondary
+            is_data_loss_allowed:
+                description:
+                - Indicates whether the operation is a global switchover or a global failover.
+                - If data loss is allowed, then the operation is a global failover. Otherwise, it is a switchover.
+                type: bool
+                sample: false
+"""
+
+
+try:
+    import botocore
+except ImportError:
+    pass  # handled by AnsibleAWSModule
+
+from ansible.module_utils.common.dict_transformations import camel_dict_to_snake_dict
+
+from ansible_collections.amazon.aws.plugins.module_utils.botocore import is_boto3_error_code
+from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule
+from ansible_collections.amazon.aws.plugins.module_utils.retries import AWSRetry
+
+
+@AWSRetry.jittered_backoff(retries=10)
+def _describe_global_clusters(client, **params):
+    try:
+        paginator = client.get_paginator("describe_global_clusters")
+        return paginator.paginate(**params).build_full_result()["GlobalClusters"]
+    except is_boto3_error_code("GlobalClusterNotFoundFault"):
+        return []
+
+
+def cluster_info(client, module):
+    global_cluster_id = module.params.get("global_cluster_identifier")
+
+    params = dict()
+    if global_cluster_id:
+        params["GlobalClusterIdentifier"] = global_cluster_id
+
+    try:
+        result = _describe_global_clusters(client, **params)
+    except (botocore.exceptions.ClientError, botocore.exceptions.BotoCoreError) as e:
+        module.fail_json_aws(e, "Couldn't get Global cluster information.")
+
```

Comment:
To be consistent with the other modules we should ensure tags parameter is always present in the returned data. However, describe_global_clusters does not return any Tags information, you should use https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/rds/client/list_tags_for_resource.html#list-tags-for-resource. You might be able to use this https://github.com/ansible-collections/amazon.aws/blob/main/plugins/module_utils/rds.py#L357

Reply:
Since no tags are linked to the global cluster, this module will not provide any tag information. I will make sure to include a note about this in the documentation for clarity.